rafx-api/src/backends/gles2/internal/conversions.rs

```diff
@@ -0,0 +1,242 @@
+            RafxFrontFace::Clockwise => gles2_bindings::CW,
+        }
+    }
+}
+
+impl RafxCompareOp {
+    pub fn gles2_compare_op(self) -> GLenum {
+        match self {
+            RafxCompareOp::Never => gles2_bindings::NEVER,
+            RafxCompareOp::Less => gles2_bindings::LESS,
+            RafxCompareOp::Equal => gles2_bindings::EQUAL,
+            RafxCompareOp::LessOrEqual => gles2_bindings::LEQUAL,
+            RafxCompareOp::Greater => gles2_bindings::GREATER,
+            RafxCompareOp::NotEqual => gles2_bindings::NOTEQUAL,
+            RafxCompareOp::GreaterOrEqual => gles2_bindings::GEQUAL,
+            RafxCompareOp::Always => gles2_bindings::ALWAYS,
+        }
+    }
+}
+
+impl RafxStencilOp {
+    pub fn gles2_stencil_op(self) -> GLenum {
+        match self {
+            RafxStencilOp::Keep => gles2_bindings::KEEP,
+            RafxStencilOp::Zero => gles2_bindings::ZERO,
+            RafxStencilOp::Replace => gles2_bindings::REPLACE,
+            RafxStencilOp::IncrementAndClamp => gles2_bindings::INCR,
+            RafxStencilOp::DecrementAndClamp => gles2_bindings::DECR,
+            RafxStencilOp::Invert => gles2_bindings::INVERT,
+            RafxStencilOp::IncrementAndWrap => gles2_bindings::INCR_WRAP,
+            RafxStencilOp::DecrementAndWrap => gles2_bindings::INCR_WRAP,
+        }
+    }
+}
+
+//TODO: Some fields in RafxRasterizerState are not handled!
+#[derive(Debug)]
+pub struct Gles2RasterizerState {
+    pub cull_mode: GLenum,
+    pub front_face: GLenum,
+    pub scissor_test: bool,
+}
+
+impl From<&RafxRasterizerState> for Gles2RasterizerState {
+    fn from(state: &RafxRasterizerState) -> Self {
+        Gles2RasterizerState {
+            cull_mode: state.cull_mode.gles2_cull_mode(),
+            front_face: state.front_face.gles2_front_face(),
+            scissor_test: state.scissor,
+        }
+    }
+}
+
+//TODO: Some fields in RafxDepthState are not handled!
+#[derive(Debug)]
+pub struct Gles2DepthStencilState {
+    pub depth_test_enable: bool,
+    pub depth_write_enable: bool,
+    pub depth_compare_op: GLenum,
+    pub stencil_test_enable: bool,
+    //pub stencil_read_mask: u8,
+    pub stencil_write_mask: u8,
+    pub front_depth_fail_op: GLenum,
+    pub front_stencil_compare_op: GLenum,
+    pub front_stencil_fail_op: GLenum,
+    pub front_stencil_pass_op: GLenum,
+    pub back_depth_fail_op: GLenum,
+    pub back_stencil_compare_op: GLenum,
+    pub back_stencil_fail_op: GLenum,
+    pub back_stencil_pass_op: GLenum,
+}
+
+impl From<&RafxDepthState> for Gles2DepthStencilState {
+    fn from(state: &RafxDepthState) -> Self {
+        Gles2DepthStencilState {
+            depth_test_enable: state.depth_test_enable,
+            depth_write_enable: state.depth_write_enable,
+            depth_compare_op: state.depth_compare_op.gles2_compare_op(),
+            stencil_test_enable: state.stencil_test_enable,
+            //stencil_read_mask: state.stencil_read_mask,
```

Comment:
This is commented out?

Reply:
yeah, it's kind of a marker for myself to look at (wrt Some fields in RafxDepthState are not handled!)